File: boj/prob02480.py
# 출처: https://www.acmicpc.net/problem/2780
# 문제: 주사위 세개

a, b, c = map(int, input().split())
k = set()
size = 0
v = 0
for i in (a, b, c):
    if i in k:
        v = i
        size += 1
        continue
    k.add(i)


if size == 2:
    print(10000 + 1000 * v)
elif size == 1:
    print(1000 + 100 * v)
else:
    print(100 * max(k))

File: boj/prob02480.java
// 출처: https://www.acmicpc.net/problem/2480
// 문제: 주사위 세개


package boj;

import java.util.Arrays;
import java.util.Scanner;

public class prob02480 {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int[] dices = new int[3];
        for (int i = 0; i < dices.length; i++) {
            dices[i] = s.nextInt();
        }
        s.close();
        Arrays.sort(dices);
        
        int res = 0;
        if (dices[0] == dices[2]) {
            res = 10000 + 1000 * dices[1];
        } else if (dices[0] == dices[1] || dices[1] == dices[2]) {
            res = 1000 + 100 * dices[1];
        } else {
            res = dices[2] * 100;
        }
        System.out.println(res);
    }
}
